src/commands/hpexpiretime.json

```
@@ -0,0 +1,85 @@
+{
+    "HPEXPIRETIME": {
+        "summary": "Returns the Unix timestamp in milliseconds since Unix epoch at which the given key's field(s) will expire",
```

This isn't a sentence
`Returns... expire`

One of these two is probably wrong.

---

I am not sure what to fix here.
Can you suggest a fix?

---

It's clunky but seems correct.

---

nit - imperative form is the common pattern? [hsetex.json](https://github.com/valkey-io/valkey/pull/2089/files#diff-251b029b5e5db50065cae387835aa82d996b3fcc16ed8c0e37ba5e9de61919e9)

```suggestion
        "summary": "Return the Unix timestamp in milliseconds since Unix epoch at which the given key's field(s) will expire",
```